modules/libcom/test/epicsCalcTest.cpp

```diff
@@ -370,6 +370,7 @@ MAIN(epicsCalcTest)
     testExpr(cosh(0.5));
     testExpr(exp(1.));
     testExpr(floor(1.5));
+    testExpr(fmod(1.5, 1.0));
```

Comment:
Check for -ve arguments as well? Not sure how C defines that.

Reply:
I've added some tests covering negative values.

fyi. [`man fmod`](https://linux.die.net/man/3/fmod) states that for `fmod(x,y)`, "... the returned value has the same sign as x ...".